### PR TITLE
Port remaining Elm parts to React

### PR DESCRIPTION
**Pull request by davidebriani**
This PR implements remaining Elm parts in React.
All Elm tooling and code is then deleted from the codebase.

### CODE REVIEW

File: src/react/Sidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  const refreshHealth = useCallback(() => {
+    appEngineHealth.refresh();
+    realmManagementHealth.refresh();
+    pairingHealth.refresh();
+    if (flowHealth) {
+      flowHealth.refresh();
+    }
+  }, [appEngineHealth, realmManagementHealth, pairingHealth, flowHealth]);
+
+  useInterval(refreshHealth, 30000);
```
Comment (Pavinati) [inline src/react/Sidebar.tsx:46]: this should be wrapped in an effect. We need to clear the interval when the sidebar is not shown (for instance, in the login page)

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 - Major rework for the Interface Editor.
 - Major rework for the Trigger Editor.
+- Major rework of Dashboard initialization, dropping Elm dependencies.
```
[the remark this answers is not in the document]
Comment (davidebriani) [reply CHANGELOG.md]: If we're listing user facing changes, I agree. I removed it

File: src/react/Sidebar.tsx (hunk 1 above)
Comment (davidebriani) [reply src/react/Sidebar.tsx:46]: It is indeed using the `useInterval` hook which clears its interval automatically

File: src/react/Router.tsx [hunk 3]
```diff
@@ -91,199 +70,142 @@ function AttemptLogin({ sessionManager }: AttemptLoginProps): React.ReactElement
...
-function DeviceDataSubPath(props: PageProps): React.ReactElement {
+function DeviceDataSubPath(): React.ReactElement {
   const { deviceId, interfaceName } = useParams();
 
-  return <DeviceInterfaceValues deviceId={deviceId} interfaceName={interfaceName} {...props} />;
-}
-
-interface NoMatchProps {
-  fallback: (requesteURL: string) => void;
+  return <DeviceInterfaceValues deviceId={deviceId} interfaceName={interfaceName} />;
```
Comment (Pavinati) [inline src/react/Router.tsx:163]: since we're using hook inside pages, we can use `useParams` hooks there aswell, removing subpaths definitions from here

**Review by Pavinati — APPROVED** [review]

**Review by bettio — APPROVED** [review]

**Pull request merged**